uiplib/GUI.py

```diff
@@ -59,45 +62,107 @@ def __init__(self, settings):
 
     def create_ui(self):
         ''' Method to initialize UI '''
-        self.container = Frame(self.root)
-        self.container.grid(row=0, column=0)
-
-        self.headerFrame = Frame(self.container)
-        self.mainFrame = Frame(self.container)
-        self.footerFrame = Frame(self.container)
-
-        self.headerFrame.grid(row=0, column=0, sticky=W+E+N)
-        self.mainFrame.grid(row=1, column=0, sticky=N+E+W+S)
-        self.footerFrame.grid(row=2, column=0, sticky=S+W+E)
-
-        self.nextButton = Button(self.mainFrame,
-                                 text=">",
-                                 command=self.next_wallpaper)
-        self.prevButton = Button(self.mainFrame,
-                                 text="<",
-                                 command=self.prev_wallpaper)
-        self.nextButton.pack(side=RIGHT, padx=5, pady=5)
-        self.prevButton.pack(side=LEFT, padx=5, pady=5)
-
-        self.downloadBtn = Button(self.footerFrame,
-                                  text="Download",
-                                  command=self.download)
-        self.setWallpaperBtn = Button(self.footerFrame,
-                                      text="Set Wallpaper",
-                                      command=self.set_wallpaper)
-        self.flushBtn = Button(self.footerFrame,
-                               text="Flush",
-                               command=self.flush)
-        self.setWallpaperBtn.pack(padx=5, pady=5)
-        self.downloadBtn.pack(padx=5, pady=5)
-        self.flushBtn.pack(padx=5, pady=5)
+        self.notebook = Notebook(self.root)
+        self.notebook.pack()
+        self.create_general_tab()
+        self.create_settings_tab()
+
+    def create_settings_tab(self):
+        self.new_settings = {}
```

Comment:
It would be good to have old settings here, and allow user to change them, giving him an idea of old settings